ffi/diplomat/ffi_coverage/src/main.rs

```diff
@@ -259,13 +263,15 @@ lazy_static::lazy_static! {
         // Not planned for 1.0
         "icu::properties::maps::CodePointMapDataBorrowed::iter_ranges",
         "icu::properties::sets::CodePointSetDataBorrowed::iter_ranges",
+        "icu::properties::script::ScriptWithExtensions::get_script_extensions_ranges",
         "icu::properties::maps::CodePointMapData::as_code_point_trie",
         "icu::properties::maps::CodePointMapData::from_code_point_trie",
         "icu::properties::sets::CodePointSetData::as_code_point_inversion_list",
         "icu::properties::sets::CodePointSetData::from_code_point_inversion_list",
         "icu::properties::sets::CodePointSetData::to_code_point_inversion_list",
         "icu::collections::codepointinvlist",
         "icu::collections::codepointtrie",
+        "icu::collections::char16trie",
```

Comment:
Question: When did we decide to exclude the three collection types? Seems like those are useful over FFI. Is there something that makes them tricky?

Reply:
I think the decision was to support building these, but I'm not sure if that's for 1.0, and I think the FFI for that is rather tricky. We can un-ignorelist, I would just rather we exposed this after 1.0 in a nicer way.